backend/src/data/models/course/course.model.ts

```diff
@@ -18,6 +18,8 @@ class Course extends Abstract {
 
   public 'courseCategoryId': number;
 
+  public 'originalId': number;
```

Comment:
But in migration you're declaring this column as `varchar`

Reply:
Oh, that's my bad
And to clarify, is it better to have it as number or string? Udemy API provides ID as numbers, but we don't know about the ones that will be added